### PR TITLE
Made controller render more efficient and added CameraConfig

### PR DESCRIPTION
**Pull request by FastestMolasses**
There was some TODO's that I implemented. For this solution, there is no need for a hashmap because we can use Bevy's query system as a hashmap. Now the function is down to O(n).

### CODE REVIEW

File: examples/minimal.rs [hunk 1]
```diff
@@ -54,36 +54,43 @@ fn setup(
...
+            },
+            FpsController {
+                air_acceleration: 80.0,
+                ..default()
+            },
+        ))
+        .insert(CameraConfig {
+            height_offset: 0.0,
+            radius_scale: 0.75,
+        })
```
Comment (FastestMolasses) [inline examples/minimal.rs:91]: This is an insert because we can't add anymore components in the bundle above.

**Comment by qhdwight** [comment]
Hi, thank you for the contribution! I will take a look this week - looks good at initial glance.

File: examples/minimal.rs (hunk 1 above)
Comment (qhdwight) [reply examples/minimal.rs:91]: Wow, did not know you could exhaust that!

Comment (qhdwight) [reply examples/minimal.rs:91]: Does this mean we should be using packages or something similar

File: src/controller.rs [hunk 2]
```diff
@@ -25,7 +25,16 @@ pub enum MoveMode {
 pub struct LogicalPlayer(pub u8);
 
 #[derive(Component)]
-pub struct RenderPlayer(pub u8);
+pub struct RenderPlayer {
+    pub id: u8,
```
Comment (qhdwight) [inline src/controller.rs:29]: I don't think there is a point to the id anymore then, right?

**Comment by qhdwight** [comment]
Thank you for the contribution!

**Review by qhdwight — APPROVED** [review]
Thank you!

**Pull request merged**